src/routes/patterns/components/Alerts.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+
+import Card from 'components/Card';
+
+export default class Alerts extends Component {
+  state = {
+    isDefault: false,
+    isDanger: false
+  };
+
+  showDefault = () => {
+    this.setState({
+      isDefault: !this.state.isDefault,
+      isDanger: false
+    });
+  };
+
+  showDanger = () => {
+    this.setState({
+      isDanger: !this.state.isDanger,
+      isDefault: false
+    });
+  };
+
+  closeAlert = () => {
+    this.setState({
+      isDefault: false,
+      isDanger: false
+    });
+  };
+
+  render() {
+    const { isDefault, isDanger } = this.state;
+
+    return (
+      <Card>
+        {isDefault &&
```

Comment:
There is a fair amount of duplication in each of these cases. I would suggest refactoring to simplify. You could utilize state-less components that just take props to help reduce the repetition. Happy to help if needed.